modules/hello-world/examples/startusing.go

```diff
@@ -1,63 +1,112 @@
 package main
 
 import (
+	"crypto/x509"
 	"fmt"
+	"io/ioutil"
+	"log"
 	"time"
 
-	gocb "github.com/couchbase/gocb/v2"
+	"github.com/couchbase/gocb/v2"
 )
 
-var bucketName = "travel-sample"
-
-// #tag::connect[]
 func main() {
-	cluster, err := gocb.Connect(
-		"localhost",
-		gocb.ClusterOptions{
-			Username: "Administrator",
-			Password: "password",
-		})
+	// Uncomment following line to enable logging
+	// gocb.SetLogger(gocb.VerboseStdioLogger())
+
+	// tag::connect[]
+	// Update this to your cluster details
+	bucketName := "travel-sample"
+	username := "Administrator"
+	password := "password"
+
+	p, err := ioutil.ReadFile("path/to/ca.pem")
 	if err != nil {
 		panic(err)
```

Comment:
are you changing all these to log.Fatal, or do they have different uses?

Reply:
`log.Fatal` is kind of similar I think, but can use change for consistency, will update.